docs/content/operations/multitenancy.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: doc_page
+---
+# Multitenancy Considerations
+
+Druid is often used to power user-facing data applications and has several features built in to better support high 
+volumes of concurrent queries.
+
+## Parallelization Model
+
+Druid's fundamental unit of computation is a [segment](../design/segments.html). Nodes scan segments in parallel and a 
+given node can scan `MIN(druid.processing.numThreads, Runtime.getRuntime().availableProcessors())` concurrently. To 
+process more data in parallel and increase performance, more cores can be added to a cluster. Druid segments 
+should be sized such that any computation over a given segment can complete in at most 500ms. By keeping computation over 
```

Comment:
Last word of this paragraph could use some re-phrasing. I'm not quite sure what you're trying to say.